pulp_rpm/app/shared_utils.py

```diff
@@ -53,3 +53,18 @@ def is_previous_revision(revision, target_revision):
         return revision == target_revision
 
     return False
+
+
+def get_key_by_value(dictionary, value):
```

Comment:
See other comment: a separate dict for lookups would be better.

Reply:
removed as separate dict used (as other comment)